iib/web/models.py

```diff
@@ -860,6 +862,10 @@ def to_json(self, verbose=True):
         rv = super().to_json(verbose=verbose)
         rv.update(self.get_common_index_image_json())
         rv['organization'] = self.organization
+        if self.omps_operator_version:
```

Comment:
You can initialize  `rv['omps_operator_version'] = {}` and then check the `if`. That way, you can remove the else block, no?

Reply:
yes, however I am used to do it this way, because it will save you some processing time (allocating memory and creating empty dictionary) in case the `if` is true.
```python
rv['omps_operator_version'] = {}  # <--- This is done every time
if self.omps_operator_version:
    rv['omps_operator_version'] = json.loads(self.omps_operator_version)
```
vs.

```python
if self.omps_operator_version:
    rv['omps_operator_version'] = json.loads(self.omps_operator_version)
else:
    rv['omps_operator_version'] = {}  # <--- This is done only in case omps_operator_version is False
```

Reply:
Yup, but it's a battle between readability and negligible processing time, in which according to me readability wins. But I'll leave the final call to you